homeassistant/components/fritz/common.py

```diff
@@ -128,6 +128,8 @@ def __init__(
         port: int = DEFAULT_PORT,
     ) -> None:
         """Initialize FritzboxTools class."""
+        super().__init__(hass=hass, logger=_LOGGER, name=f"{DOMAIN}-coordinator")
```

Comment:
```suggestion
        super().__init__(hass=hass, logger=_LOGGER, name=f"{DOMAIN}-{host}-coordinator")
```
to differentiate when two or more Fritz!Box are integrated

Reply:
Thx!